drivers/wifi/esp_at/Kconfig.esp_at

```diff
@@ -144,6 +144,11 @@ config WIFI_ESP_AT_VERSION_2_0
 	help
 	  Use AT command set version 2.0.
 
+config WIFI_ESP_AT_VERSION_2_1
+	bool "AT version 2.1"
+	help
+	  Use AT command set version 2.1.
+
```

Comment:
I think we should default to this option, as this is the newest firmware version we support.